content/tutorials/themes/theme-objects/breadcrumb/index.md

```diff
@@ -11,11 +11,11 @@ links:
 
 # Breadcrumb Theme Object Introduction
 
-The Breadcrumb Theme Object displays the path to the currently selected page in the menu structure like this:
-RootPage > SubPage > SubSubPage
-The starting level, separator and styliong are configurable.
+The Breadcrumb Theme Object displays the path to the currently selected page in the menu structure like this:/n
```

Comment:
```suggestion
The Breadcrumb Theme Object displays the path to the currently selected page in the menu structure like this:
```

Reply:
Hmm, I think it's much more readable with the breaks.. BTW if you look at the current source you will see that there are already breaks in the text and both Github preview and the dnndocs site ignore them..

Reply:
Breaks are not necessary as long as the text is on separate lines.  The document GitHub does not have separate links.  You may be seeing them translate to multiline in the diff tool, but the raw file does not have separate lines.  I agree it looks better on separate lines.  ;-)

Reply:
Hmm, right i see now that there's \n as text in the source, I'll correct that. Not so smart that I missed that..

Reply:
No worries, I just committed my suggestion.  Thanks.

Reply:
ok thank you